USAGE_android.md

```diff
@@ -833,6 +834,8 @@ optional arguments:
                         setup for replay. The default without this option is to use a Virtual Swapchain
                         of images which match the swapchain in effect at capture time and which are
                         copied to the underlying swapchain of the implementation being replayed on.
+  --offscreen           Disable creating VkSurface, VkSwapchain and a window. If users want to see      
```

Comment:
We decided not to output rendering images because the features are duplicated with screenshots. If the users want to see the images, they need to add screenshots options. I added this in the fourth commit.

Reply:
Modified.